software/opensign.yml

```
@@ -0,0 +1,16 @@
+name: "OpenSign™"
+website_url: "https://www.opensignlabs.com"
+source_code_url: "https://github.com/opensignlabs/opensign"
+description: "Free, open source & self-hosted DocuSign alternative."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Nodejs
+  - MongoDB
+  - Docker
+tags:
+  - Document Management
+  - Digital Signature
+  - e-signature
+  - DocuSign alternative
+  - Free digital signatures
```

Tags are predefined and can be found here: https://github.com/awesome-selfhosted/awesome-selfhosted-data/tree/master/tags
Depends on 3rd party due to MongoDB

```suggestion
tags:
  - Document Management
depends_3rdparty: true
```